pootle/apps/virtualfolder/helpers.py

```diff
@@ -27,7 +27,7 @@ def make_vfolder_treeitem_dict(vfolder_treeitem):
         'icon': 'vfolder'}
 
 
-def extract_vfolder_from_path(request_path):
+def extract_vfolder_from_path(request_path, vfti=None):
```

Comment:
Update the docstring below to include the new argument. Don't forget to specify it should be a VirtualFolderTreeItem queryset.